source/guide/3.2/plugin-vcloud.md

```diff
@@ -124,46 +124,48 @@ Two additional runtime-properties are available on node instances of this type o
   * `cloudify.interfaces.lifecycle.creation_validation` validates Port node parameters
 
 
-## cloudify.vcloud.nodes.FloatingIP
+## cloudify.vcloud.nodes.PublicNAT
 
 **Derived From:** [cloudify.nodes.VirtualIP](reference-types.html)
 
 **Properties:**
 
-* `floatingip` key-value floating ip configuration.
+* `nat` key-value NAT configuration.
     * `edge_gateway` vCloud gateway name
     * `public_ip` public ip. If not specified public ip will be allocated from the pool of free public ips.
+* `rules` list of NAT rules configurations.
+    * `type` NAT types. Can be 'SNAT' or 'DNAT'.
+    * `protocol` network protocol. Can be 'tcp', 'udp', 'tcpudp' or 'any'. Applies only for 'DNAT'.
+    * `original_port` original port. Applies only for 'DNAT'.
+    * `translated_port` translated port. Applies only for 'DNAT'.
+* `use_external_resource` a boolean for setting whether to create the resource or use an existing one. Defaults to `false`.
 * `vcloud_config` see the [vCloud Configuration](#vcloud-configuration).
 
 **Mapped Operations:**
 
-  * `cloudify.interfaces.lifecycle.creation_validation` validates FloatingIP node parameters
+  * `cloudify.interfaces.lifecycle.creation_validation` validates PublicNAT node parameters
 
 **Attributes:**
 
   * `public_ip` public ip address
 
-
-## cloudify.vcloud.nodes.PublicNAT
+## cloudify.vcloud.nodes.FloatingIP
```

Comment:
have you considered, instead of having this type, having the PublicNAT type's properties' defaults be set in a way so that the default would be the same as the FloatingIP for simplified use?